FrontEnd/src/pages/Auth.tsx

```diff
@@ -0,0 +1,95 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { useAuthStore } from "../store/authStore";
+import LoginForm from "../components/LoginForm";
+import SignupForm from "../components/SignUpForm";
+
+export default function Login() {
+  const navigate = useNavigate();
+  const login = useAuthStore((state) => state.login);
+  const signup = useAuthStore((state) => state.signup);
+  const [isSignUp, setIsSignUp] = useState(false);
+
+  const handleSubmitLogin = (email: string, password: string) => {
+    login(email, password);
+    navigate("/profile");
```

Comment:
navigation should depends on the value that returned from the login/signup function.